### PR TITLE
manifest.md: Update fields and explanations

### PR DESCRIPTION
**Pull request by r0b5t4**

### CODE REVIEW

**Comment by r0b5t4** [comment]
@baleboy ptal

File: manifest.md [hunk 1]
```diff
@@ -1,36 +1,18 @@
...
-* Extension field `xwalk_windows_update_id`: Automatically generated identifier for native windows support (work in progress).
-
-* Extension field `xwalk_windows_vendor`: Vendor string for native windows support (work in progress).
+* `display`:	http://w3c.github.io/manifest/ Values "standalone" and "fullscreen" are supported on Android.
+* `icons`:	http://w3c.github.io/manifest/ Basic support, only one resolution per icon entry.
+* `name`:	http://w3c.github.io/manifest/
+* `orientation`:	http://w3c.github.io/manifest/
+* `short_name`:	http://w3c.github.io/manifest/ Value has no effect on Android, as `name` is used for the launcher label.
+* `start_url`:	http://w3c.github.io/manifest/
+* `xwalk_android_animatable_view: true/false`: Whether to allow pinch zooming.
```
Comment (baleboy) [inline manifest.md]: this is also about using a textureView (true) or SurfaceView (False) as rendering backend

File: manifest.md [hunk 2]
```diff
@@ -1,36 +1,18 @@
...
+* `xwalk_android_keep_screen_on: true/false`: Whether to keep the screen on while app is in the foreground.
+* `xwalk_android_permissions: <array-of-permissions>`: Android manifest permissions to add to default ones. Names without prefix, e.g. `[AUDIO, CAMERA]`, see http://developer.android.com/reference/android/Manifest.permission.html.
```
Comment (baleboy) [inline manifest.md]: why do we need this, given that we have xwalk_permissions?

**Comment by baleboy** [comment]
Can you call out what parameters are mandatory, e.g. xwalk_package_id

File: manifest.md (hunk 1 above)
Comment (r0b5t4) [inline manifest.md]: Yes, and currently broken. I wanted to add a note about that. https://crosswalk-project.org/jira/browse/XWALK-4692

File: manifest.md (hunk 2 above)
Comment (r0b5t4) [inline manifest.md]: My bad, I was unaware about xwalk_permissions when I implemented it. It is mostly for internal use, to accumulate extra permissions required by extensions. We can leave it undocumented/unsupported, and disable it for the next release.

File: manifest.md (hunk 1 above)
Comment (baleboy) [inline manifest.md]: that bug is about launch screen?

Comment (r0b5t4) [inline manifest.md]: sorry https://crosswalk-project.org/jira/browse/XWALK-4681

File: manifest.md [hunk 3]
```diff
@@ -1,36 +1,22 @@
...
+
+* `name`:	http://w3c.github.io/manifest/ Will be uses as label in the android launcher.
+* `icons`:	http://w3c.github.io/manifest/ Basic support, only one resolution per icon entry.
+* `start_url`:	http://w3c.github.io/manifest/ First html page to load.xw
+* `xwalk_app_version: "a.b.c"`: Three part application version number e.g. 1.2.3 needed for android and windows packaging.
+* `xwalk_package_id: "com.example.foo"`: Canonical name e.g. com.example.foo.
+
+### Extra fields
+
+* `display`:	http://w3c.github.io/manifest/ Values "standalone" and "fullscreen" are supported on Android.
```
Comment (baleboy) [inline manifest.md]: I believe they are the only fields supported on any platform (at least Windows), make it general?

File: manifest.md [hunk 4]
```diff
@@ -1,36 +1,22 @@
...
+* `orientation`:	http://w3c.github.io/manifest/
+* `short_name`:	http://w3c.github.io/manifest/ Value has no effect on Android, as `name` is used for the launcher label.
+* `xwalk_android_animatable_view: true/false`: Whether to allow pinch zooming, currently out of order: https://crosswalk-project.org/jira/browse/XWALK-4681.
```
Comment (baleboy) [inline manifest.md]: How about you remove this from the doc until it works?

File: manifest.md (hunk 3 above)
Comment (r0b5t4) [inline manifest.md]: The doc is just for android now, as that is the major target for this release.

File: manifest.md (hunk 4 above)
Comment (r0b5t4) [inline manifest.md]: done

**Comment by baleboy** [comment]
LGTM

**Comment by r0b5t4** [comment]
squash merged

**Pull request closed without merging**